tests/helper/component_podman.go

```diff
@@ -0,0 +1,75 @@
+package helper
+
+import (
+	"fmt"
+	"os/exec"
+
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	jsonserializer "k8s.io/apimachinery/pkg/runtime/serializer/json"
+	"k8s.io/kubectl/pkg/scheme"
+)
+
+// PodmanComponent is an abstraction for a Devfile Component deployed on podman
+type PodmanComponent struct {
+	name string
+	app  string
+}
+
+func NewPodmanComponent(name string, app string) *PodmanComponent {
+	return &PodmanComponent{
+		name: name,
+		app:  app,
+	}
+}
+
+func (o *PodmanComponent) ExpectIsNotDeployed() {
+	podName := fmt.Sprintf("%s-%s", o.name, o.app)
+	cmd := exec.Command("podman", "pod", "list", "--format", "{{.Name}}", "--noheading")
```

Comment:
I'm not sure. This is used during integration tests only, on a stable environment, I cannot see a reason for the moment to have to configure it

Reply:
I agree from the environment perspective.
I thought it would make sense for an `odo` dev to make sure the same podman binary is used when running the actual `odo` commands in the tests and when performing the assertions here..
But okay, that's something that can be optimized later on.
/lgtm